packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,146 @@
+import HomeIcon from "@mui/icons-material/Home";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import Stack from "@mui/material/Stack";
+import { styled } from "@mui/material/styles";
+import React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import StyledTooltip from "./OSCALStyledTooltip";
+import {
+  OSCALSystemImplementationTableTitle,
+  StyledHeaderTableCell,
+  StyledTableRow,
+  StyledTableHead,
+} from "./OSCALSystemImplementationTableStyles";
+
+const OSCALPropertiesButton = styled(Button)(
+  ({ theme }) => `
+    color: ${theme.palette.primary.main};
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+  `
+);
+
+function OSCALPropertiesCard(props: any) {
+  const { title, children } = props;
+
+  const [open, setOpen] = React.useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <OSCALPropertiesButton
+        size="small"
+        variant="outlined"
+        onClick={handleOpen}
+        aria-label={`${title} properties button`}
+        startIcon={<HomeIcon />}
+      >
+        Properties
+      </OSCALPropertiesButton>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        scroll="paper"
+        aria-labelledby="scroll-dialog-title"
+        aria-describedby="scroll-dialog-description"
+        maxWidth="md"
+        fullWidth
+      >
+        {children}
+      </Dialog>
+    </>
+  );
+}
+
+function isNistNamespace(namespace: string) {
+  return !namespace || namespace.includes("nist.gov");
+}
```

Comment:
Consider trying to use something from `OSCALPropUtils` (exporting something that exists or adding a new function)

Reply:
This is a little different than what is in `OSCALPropUtils`, more of a validation check rather than providing a NIST namespace. I've gone ahead and added this to `OSCALPropUtils`.